cartographer/mapping_3d/local_trajectory_builder.cc

```diff
@@ -158,7 +158,7 @@ LocalTrajectoryBuilder::AddAccumulatedRangeData(
     return nullptr;
   }
   ceres_scan_matcher_->Match(
-      matching_submap->local_pose().inverse() * pose_prediction,
+      (matching_submap->local_pose().inverse() * pose_prediction).translation(),
```

Comment:
Can we write this more efficiently now?

Reply:
Done

Reply:
Actually, I reverted the more efficient computation because it was less readable.